solvation_analysis/tests/test_basics.py

```diff
@@ -0,0 +1,87 @@
+import sys
```

Comment:
If I remember right, I think if you call this file "conftest.py" the fixtures are magically picked up by pytest (and won't need importing)

Reply:
Wow that is magic!